lib/os/mpsc_pbuf.c

```diff
@@ -201,10 +201,12 @@ static bool drop_item_locked(struct mpsc_pbuf_buffer *buffer,
 
 	uint32_t rd_wlen = buffer->get_wlen(item);
 
-	/* If packet is busy need to be ommited. */
+	/* If packet is busy need to be omitted. */
 	if (!is_valid(item)) {
 		return false;
 	} else if (item->hdr.busy) {
```

Comment:
No need for `else if` as the block ends with a `return`.

```suggestion
	}
	if (item->hdr.busy) {
```

Reply:
I thought about commenting there as well, but it's not saving a level of indentation, so maybe that drive-by isn't totally needed 🤷🏼‍♂️